vllm_ascend/quantization/quantizer.py

```diff
@@ -39,12 +49,13 @@ def get_quantizer(cls,
         try:
             module = importlib.import_module("mindie_turbo")
             MindIETurboQuantizer = module.MindIETurboQuantizer
+            return MindIETurboQuantizer.get_quantizer(quant_config, prefix,
+                                                      packed_modules_mapping)
         except ImportError:
-            raise NotImplementedError(
-                "There is no available ascend quantizer.")
-
-        return MindIETurboQuantizer.get_quantizer(quant_config, prefix,
-                                                  packed_modules_mapping)
+            logger.info(
+                "MindIE Turbo not available, using the adapted version now!")
```

Comment:
This dumps 40x line repeat logs when launch model:
```
INFO 04-19 15:07:35 [quantizer.py:55] MindIE Turbo not available, using the adapted version now!
```

Reply:
I move this to `VLLMAscendQuantizer.__init__`
<img width="969" alt="image" src="https://github.com/user-attachments/assets/d0d6eef2-96cf-44f8-aa40-dda6f370bdd9" />